code/datums/namepool.dm

```diff
@@ -35,3 +35,9 @@ GLOBAL_LIST_EMPTY_TYPED(namepool, /datum/namepool)
 	firstname_male_pool = "names/moth_first"
 	firstname_female_pool = "names/moth_first"
 	lastname_pool = "names/moth_last"
+
+/datum/namepool/sectoid
+	lastname_pool = "names/sectoid_last"
+
+/datum/namepool/sectoid/get_random_name()
+	return "Sectoid [pick(ascii2text(rand(49, 57)))][pick(ascii2text(rand(65, 90)))][pick(ascii2text(rand(65, 90)))]"
```

Comment:
instead of this incomprehensible code maybe just use pick()
`pick("A","B","C","D","E","F")`